docs/testplan/Smartswitch-test-plan.md

```diff
@@ -645,6 +672,673 @@ root@sonic:/home/cisco# show chassis modules status
  *  Verify number of DPUs from inventory file for the testbed and number of DPUs shown in the cli output.
 
 
+### 1.13 Check Memory on DPU
+
+### Steps
+
+ * Infrastructure support will be provided to choose from following clis or in combinations based on the vendor.
+ * Use command `show system-memory` to get memory usage on each of those DPUs
+ * Use `show system-health dpu <DPU_NUM>` to check memory check service status
```

Comment:
Is this command to be run on DPU or NPU? If it is NPU, could you clarify it here?

Reply:
Also, when you have mix of commands to run on NPU and DPU, could you please clarify where you are running the command in the text in all the tests?

Reply:
It is on DPU. 

Yes I understand that it is confusing as we jump between npu and dpu. 
I tried to mention that to an extent on each cases on where it should be running. 

- I have the section called verify in to tell explicitly on where to run in all the cases. 
- In sample output as well, I mentioned before on where it is running. 
- On title also for all dpu run cases, I have titled as on dpu.

Reply:
As some of the tests are having commands to check in both NPU and DPU. Could you add "NPU:" or "DPU:" prefix before mentioning the command to use? Will it be more clear?